dts/bindings/pinctrl/bflb,bl-pinctrl.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2021-2024 Gerson Fernando Budke <nandojve@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Bouffalo Lab Pinctrl node
+
+compatible: "bflb,bl-pinctrl"
+
+include: base.yaml
+
+properties:
+  "#address-cells":
+    required: true
+    const: 1
+  "#size-cells":
+    required: true
+    const: 1
+
+child-binding:
+  description: |
+    Bouffalo Lab pin controller pin configuration nodes. Each node is composed
+    by one or more groups, each defining the configuration for a set of pins.
+
+  child-binding:
+    description: |
+      Bouffalo Lab pin controller pin configuration group. Each group contains
+      a list of pins sharing the same set of properties. Example:
+      uart0_default: uart0_default {
+        /* group 1 (name is arbitrary) */
+        pins1 {
+          /* configure to uart0 function plus modem interrupt, pin 7 as UART_RX
+             pin 16 as UART_TX and finally pin 18 as gpio */
+          bflb,pins = <BFLB_PIN(uart0,   7)>,
+                      <BFLB_PIN(uart0,  16)>,
+                      <BFLB_PIN(gpio,   18)>;
+          bflb,signals = <BFLB_SIG_UART_RXD>,
+                         <BFLB_SIG_UART_TXD>;
+          bias-pull-up;
+          input-schmitt-enable;
+        };
+      };
+      The list of supported standard properties:
+      - bias-high-impedance: Disable pull-up/down (default behavior, not
+        required).
+      - bias-pull-up: Enable pull-up resistor.
+      - bias-pull-down: Enable pull-down resistor.
+      - input-enable: Enable GPIO as input (default behavior, not required).
+      - input-schmitt-enable: Enable Schimitt Trigger when GPIO is Input.
+      - output-enable: Enable GPIO as output.
+
+      Note that bias options are mutually exclusive. It is the same with GPIO
+      input/output enable options.
+
+    include:
+      - name: pincfg-node.yaml
+        property-allowlist:
+          - bias-high-impedance
+          - bias-pull-down
+          - bias-pull-up
+          - input-enable
+          - input-schmitt-enable
+          - output-enable
+
+    properties:
+      pinmux:
+        required: true
+        type: array
+        description: |
+          An array of pins sharing the same group properties. The pins should be
+          defined using the BFLB_PINMUX utility macro that encode all the pin
+          route matrix.
+      drive-strength:
+        type: int
+        default: 0
+        enum:
+          - 0     # Default value, lower strength, 8mA
+          - 1     # 9.6mA
+          - 2     # 11.2mA
+          - 3     # highest strength, 12.8mA
```

Comment:
add commeentgs to desc. default needs justification

Reply:
Added the following justification:
_The driver will automatically apply the default value (8mA) to all pins to save power._